src/qibo/transpiler/unroller.py

```diff
@@ -22,7 +22,11 @@ class FlagMeta(EnumMeta):
 
     def __getitem__(cls, keys):
         if isinstance(keys, str):
-            return super().__getitem__(keys)
+            try:
+                return super().__getitem__(keys)
+            except KeyError:
+                log.info(f"Native gate {keys} is not supported by qibo.")
```

Comment:
I know we're inconsistent in logging, but in this respect I became strongly opinionated.

https://github.com/qiboteam/qibo-cloud-backends/pull/30#discussion_r1820379326

Reply:
Should we remove the log?

Reply:
[23423a9](https://github.com/qiboteam/qibo/pull/1504/commits/23423a9b0b45d5acddd839eb19ceac3197c82954)